packages/core/src/new-api/types/module.ts

```diff
@@ -361,6 +361,15 @@ export interface ModuleParameters {
   [parameterName: string]: ModuleParameterType;
 }
 
+/**
+ * An object containing a map of module ID's to their respective ModuleParameters.
+ *
+ * @beta
+ */
+export interface DeploymentParameters {
```

Comment:
I think we should move this to `./types/deployer.ts`, as that is where the user will encounter it first.

What do you think?

Reply:
Makes sense to me